src/envoy/mixer/envoy.conf.template

```diff
@@ -65,6 +120,17 @@
             "url": "tcp://${BACKEND}"
           }
         ]
+      },
+      {
+        "name": "service2",
+        "connect_timeout_ms": 5000,
+        "type": "strict_dns",
+        "lb_type": "round_robin",
+        "hosts": [
+          {
+            "url": "tcp://localhost:9090"
+          }
+        ]
```

Comment:
this is the actual config in JSON. not a JSON schema template.  the reason it is called "template" since it has ${MIXER_SERVER} variables,  config gen will replace them with real mixer_server address.